apps/namadillo/src/atoms/fees/atoms.ts

```diff
@@ -83,8 +83,10 @@ export const gasPriceTableAtom = atomWithQuery<GasPriceTable>((get) => {
         const baseAmount = BigNumber(minDenomAmount);
         return {
           token,
-          // TODO should we run `toDisplayAmount` for all tokens or only NAM?
-          gasPrice: asset ? toDisplayAmount(asset, baseAmount) : baseAmount,
+          gasPrice:
+            asset && isNamadaAsset(asset) ?
```

Comment:
so is this the answer of the issue!? 🫠

Reply:
yep :/ we're receiving unam from the indexer, and we use nam everywhere, even on the cli, differently of other tokens